adhocracy4/projects/migrations/0043_migrate_topics_to_m2m_topics.py

```diff
@@ -1,19 +1,25 @@
 # Generated by Django 4.2 on 2023-11-29 13:20
+import sys
+import importlib
 
 from django.db import migrations
 from django.conf import settings
 
 
 def add_topics_to_m2m_table(apps, schema_editor):
     if hasattr(settings, "A4_PROJECT_TOPICS"):
-        topicsenum = settings.A4_PROJECT_TOPICS
+        # get the module from the settings string path without the class
+        index = settings.A4_PROJECT_TOPICS.rfind(".")
+        # get the class name from the settings string path
+        module = importlib.import_module(settings.A4_PROJECT_TOPICS[:index])
+        topicsenum = getattr(module, settings.A4_PROJECT_TOPICS[index + 1 :])
```

Comment:
This seems a bit complicated and raises the question whether we need the `name` of the topic in the database. For our purposes the `code` would be enough, right?

And if we need the label (e.g. in the template or via the api) we can get it via `topicsenum(code).label`, for example in the serializer or by overwriting the `def _str_` of the Topic model.

Reply:
I agree, we don't need the name. I will remove it then.